Firestore/Source/API/FIRFirestore.mm

```diff
@@ -175,9 +175,9 @@ - (void)setSettings:(FIRFirestoreSettings *)settings {
 }
 
 /**
- * Ensures that the FirestoreClient is configured and returns it.
+ * Returns the FirestoreClient if it is configured.
  */
-- (FSTFirestoreClient *)client {
+- (nullable FSTFirestoreClient *)client {
```

Comment:
Is this actually desirable? The comment was wrong, but should anyone be accessing this before it's set up?

Why not assert that the client is non-null and then keep the nullability the way it was?

Reply:
Good question.  Thanks for prodding me to be more holistic. :-)  I ended up noticing that this is barely used and so I deleted it, but added the assert you wanted to `api::Firestore.client()` which was also missing it.